wpilib/wpilib/speedcontrollergroup.py

```diff
@@ -7,15 +7,35 @@
 # the project.
 # ----------------------------------------------------------------------------
 from .interfaces.speedcontroller import SpeedController
+from .sendablebase import SendableBase
 
 __all__ = ["SpeedControllerGroup"]
 
 
-class SpeedControllerGroup(SpeedController):
+class SpeedControllerGroup(SendableBase, SpeedController):
+    """Allows multiple :class:`.SpeedController` objects to be linked together"""
+
+    instances = 0
+
     def __init__(self, *args):
+        """Create a new SpeedControllerGroup with the provided SpeedControllers.
+
+        :param args: SpeedControllers to add
+        :type args: :class:`.SpeedController`
+        """
+        SendableBase.__init__(self)
+        SpeedController.__init__(self)
+
+        for speedController in args:
+            self.addChild(speedController)
         self.speedControllers = args
         self.isInverted = False
 
+        self.instances += 1
+        self.setName("SpeedControllerGroup", self.instances)
+
+
+
```

Comment:
3 blank lines? That must be a record.

(You have a couple of 2 blank lines in other places, but I'm not going to spam you with review comments.)

Reply:
Oh dustin is away isn't he...that means I can format my files